src/drivers/mpu6000/mpu6000.cpp

```diff
@@ -658,19 +658,19 @@ MPU6000::init()
 	_accel_reports = new ringbuffer::RingBuffer(2, sizeof(accel_report));
 
 	if (_accel_reports == nullptr) {
-		goto out;
+		return ret;
```

Comment:
it's the jumping over the new declarations.  But `out` do not thing of value so it is all good.